src/dbnode/storage/bootstrap/bootstrapper/commitlog/options.go

```diff
@@ -34,7 +34,7 @@ const (
 	// DefaultReturnUnfulfilledForCorruptCommitLogFiles is the default
 	// value for whether to return unfulfilled when encountering corrupt
 	// commit log files.
-	DefaultReturnUnfulfilledForCorruptCommitLogFiles = true
+	DefaultReturnUnfulfilledForCorruptCommitLogFiles = false
```

Comment:
why is this the default we want?

Reply:
Added to commit description the link to the corresponding documentation:
https://github.com/m3db/m3/blob/64c6dea950ce27108841ecb82304cf8c46eea158/site/content/docs/operational_guide/availability_consistency_durability.md#ignoring-corrupt-commitlogs-on-bootstrap